Curs04/src/views/common/common.js

```diff
@@ -26,10 +27,10 @@ const NAV = function (options={}) {
                 </div>
                 <div class="col-md-4 evaluation-button" style="padding-top: 4px;">
                     <!--Evaluation navbar button-->
-                    <a class="border-bottom" href="#">Evaluations</a>
+                    <a id="evaluation-page" class="border-bottom" href="#">Evaluations</a>
```

Comment:
the name is not really relevant to what it represents here; this button cannot represent the `evaluation-page`, can it?